### PR TITLE
Add cityhash to requirements.txt

### PR DESCRIPTION
**Pull request by bmerry**
Required for ska-sa/katdal#289.

### CODE REVIEW

**Comment by bmerry** [comment]
I pinned the version because this repo is using the legacy (Python 2) branch of katsdpdockerbase, so doesn't benefit from ska-sa/katsdpdockerbase#45.

File: requirements.txt [hunk 1]
```diff
@@ -5,6 +5,7 @@ boto==2.48.0
 botocore
 certifi
 chardet
+cityhash==0.2.3post9
```
Comment (ludwigschwardt) [inline requirements.txt]: Why is this one pinned?

Comment (bmerry) [reply requirements.txt]: See my [prior comment](https://github.com/ska-sa/katsdpdata/pull/161#issuecomment-591368381).

Comment (ludwigschwardt) [reply requirements.txt]: Missed that, sorry!

**Pull request merged**